TASVideos/Pages/Forum/Subforum/Index.cshtml

```diff
@@ -16,34 +16,34 @@
 </div>
 <hr />
 <partial name="_Pager" model="Model.Forum.Topics" />
-
-<table class="table table-striped table-bordered">
```

Comment:
did tabs change to spaces here? if so, we want tabs, hard to tell with the indenting changing